wp-content/themes/core/components/blocks/gallery_grid/gallery_grid.php

```diff
@@ -0,0 +1,64 @@
+<?php
+declare( strict_types=1 );
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = \Tribe\Project\Templates\Components\blocks\gallery_grid\Gallery_Grid_Controller::factory( $args );
+?>
+
+<section <?php echo $c->get_classes(); ?> <?php echo $c->get_attributes(); ?>>
+	<div <?php echo $c->get_container_classes(); ?>>
+
+		<div class="b-gallery-grid__header">
+			<?php if ( ! empty( $c->get_header_args() ) ) { ?>
```

Comment:
Curious why you're wrapping the content block in another container? You can apply the `b-gallery-grid__header` class to the content block (also, would be good to make it a html `<header>` if it's not), and you can use the content blocks' CTA field as the slideshow button rather than appending it after the fact. (less markup for the same effect.)